src/acquisition/covidcast/database.py

```diff
@@ -612,5 +622,21 @@ def update_covidcast_meta_cache(self, epidata_json):
         `timestamp` = UNIX_TIMESTAMP(NOW()),
         `epidata` = %s
     '''
+    epidata_json = json.dumps(metadata)
 
     self._cursor.execute(sql, (epidata_json,))
+
+  def retrieve_covidcast_meta_cache(self):
```

Comment:
A docstring should probably be added. Additionally this method doesn't seem to be used.

Reply:
i used it in debugging...  i can remove it if you think thats best practices or whatever.

Reply:
This seems like a useful function. Probably we only need to add a docstring and keep it.